packages/i18n-translate/src/index.js

```diff
@@ -0,0 +1,82 @@
+import _ from 'lodash';
+import { sprintf } from 'sprintf-js';
+
+const pluralizeFunctions = {
+  en: number => (number === 0 || number > 1 ? 'other' : 'one'),
+  fr: number => (number > 1 ? 'other' : 'one'),
+  hu: (number, general) => {
+    if (!general) {
+      return 'one';
+    }
+
+    return number > 1 ? 'other' : 'one';
+  },
+  hr: (number, general) => {
+    // General plural
+    if (general) {
+      return number > 1 ? 'other' : 'one';
+    }
+
+    const numberInString = number.toString();
+    const lastDigit = numberInString.charAt(numberInString.length - 1);
+
+    if ((number > 4 && number < 21) || ['0', '5', '6', '7', '8', '9'].includes(lastDigit)) {
+      // Third plural form
+      return 'many';
+    } else if (lastDigit === '1') {
+      // First plural form and singular
+      return 'one';
+    } else if (lastDigit === '2' || lastDigit === '3' || lastDigit === '4') {
+      // Second plural form
+      return 'few';
+    }
+
+    return '';
+  },
+};
+
+export const translate = (lang, i18nNames = {}, errorCallback = _.noop, renderersHandler = _.noop) => {
+  const pluralize = pluralizeFunctions[_.get(lang, '_i18n.lang')] || pluralizeFunctions.fr;
+
+  return (key, data = {}, number, general, renderers) => {
```

Comment:
We could run a null-safety check in a closure providing the resulting function with `lang` value that is always something meaningful, i.e.:
```suggestion
  // don't mind the name
  const notNullOrUndefinedLang = lang || {}
  return (key, data = {}, number, general, renderers) => {
```
Then we could safely use this value without redundant `_get` call since we already sure it's not `null` or `undefined`.